composables/autoTeleport/useAutoTeleportTransitionDetails.ts

```diff
@@ -157,9 +179,9 @@ export default function (
     return sourceChainProperties?.tokenSymbol === chainSymbol.value
   })
 
-  const fetchTeleportFee = computed(
+  const fetchTeleportFee = computed<boolean>(
```

Comment:
confusing computed name, maybe use something like `hasTeleportFee` or `hasFee`

Reply:
changed it to canGetTeleportFee not sure if any better , don't think hasTeleportFee or hasFee  explain well what that computed does wdyt